packages/website/utils/content.ts

```diff
@@ -86,18 +44,57 @@ async function fetchFiles(targetPath: string) {
   return fetchedFiles;
 }
 
-async function geltAllMDX() {
-  const allMDX = await fetchFiles(componentsPath);
-  const mdxData = allMDX.map((filepath) => getSlugByFilepath(filepath));
+async function getAllMdx(paths: string[]) {
+  let allMDX: string[] = [];
 
-  // Saving a map of slugs and the filepath to their mdx in a JSON file
-  const mdxFilepathBySlug = mdxData.reduce((acc, mdx) => {
-    return { ...acc, [mdx.slug]: mdx.filepath };
-  }, {});
-  const data = JSON.stringify(mdxFilepathBySlug, null, 2);
-  fs.writeFileSync('utils/mdxFilepathBySlug.json', data);
+  for (const path of paths) {
+    const newFiles = await fetchFiles(path);
+    allMDX = allMDX.concat(newFiles);
+  }
```

Comment:
I think I didn't understand this change

Why do we need to initiate an empty array and then concat the fetched files?
Instead of just fetching the files directly like it was before

Reply:
it's fetching from several folders, not from one only

Reply:
Ahhh you mean that now it fetches from `/components` and `/core`

got it!

And we need to concat because both "sources" will return an array of files, right?

Reply:
yep